.circleci/config.yml

```diff
@@ -74,6 +100,18 @@ jobs:
       - *run_yarn
       - run: yarn test --maxWorkers=2
 
+  test_source_experimental:
```

Comment:
I assume that since we're using parallel containers, this won't actually increase the duration of every CI job? That would not be great.

Reply:
The workflows run in parallel, yeah